README.md

```diff
@@ -2,36 +2,47 @@
 
 # Zammad admin documentation
```

Comment:
Hm, we might want to capitalize this too.
But OTOH, if you don't use capitalization style consistently yet in the docs, we can keep it like it is for now.

Reply:
"Title case" that is. https://capitalizemytitle.com/

Reply:
No it's a good point and addressed already. :)